setup.scripts/mailscanner/00_mw-install-script.conf

```diff
@@ -0,0 +1,4 @@
+#Default config provided by mailwatch-install-script
+#Do NOT edit this file. It is auto generated.
+#Instead create your own config that arranged after this file in alphabetical order.
+Virus Scanners = clamav
```

Comment:
All these needed to have clamav running:

```
Virus Scanners = clamd
Clamd Socket = /var/run/clamav/clamd.ctl
Clamd Lock File = /var/run/clamav/clamd.pid
Clamd Use Threads = no
```

Reply:
If you use `clamav` its not necessary. All you need to do is define it as `Virus Scanners`.
Keep in mind that there is a difference between clamav and clamd:
https://github.com/MailScanner/v5/blob/master/common/etc/MailScanner/virus.scanners.conf

Reply:
Was not working with just "Virus Scanners = clamav"
I restart a complete fresh try. Will see.

Reply:
Hmm. I have a server running with default config and its scanning my mails with it.
Have you tried sending an EICAR file to test?

Reply:
OK, it work with last try but its very slow compared to clamd.

Try from GUI with ms_lint.php:

===========================================================================	5.0E-5
Filename Checks: Windows/DOS Executable (1 eicar.com)	0.35477
Other Checks: Found 1 problems	0.05216
Virus and Content Scanning: Starting	0.00054
1.message: Eicar-Test-Signature FOUND	37.38406
./1/eicar.com: Eicar-Test-Signature FOUND	0.00392
Virus Scanning: ClamAV found 2 infections	1.2392
Infected message 1 came from 10.1.1.1	0.00139
Virus Scanning: Found 2 viruses	0.0009
===========================================================================	0.0141
Virus Scanner test reports:	4.0E-5
ClamAV said "eicar.com contains Eicar-Test-Signature"	0.00028
If any of your virus scanners (clamav)

Reply:
Was running during ms_lint.php

www-data 27735 26303  0 11:54 ?        00:00:00 sh -c sudo /usr/sbin/MailScanner --lint 2>&1
postfix  27780 27737  0 11:54 ?        00:00:00 /bin/sh /usr/lib/MailScanner/wrapper/clamav-wrapper /usr -r --infected --stdout .
postfix  27791 27780 99 11:54 ?        00:00:18 /usr/bin/clamscan --tempdir=/var/spool/MailScanner/incoming/clamav-tmp/tmp.y3FUMlcaE0 -r --infected --stdout .